<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddIncomeDialog</class>
 <widget class="QDialog" name="AddIncomeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddIncomeDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(35, 39, 42);

</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>431</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(35, 39, 42);

border:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="incomeNameLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whitney</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit[text]{
color: white;}
QLineEdit{
background-color: rgb(44, 47, 51);
border:none;}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Income name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="incomeValue">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whitney</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit[text]{
color: white;}
QLineEdit{
background-color: rgb(44, 47, 51);
border:none;}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cardsIncomeComboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whitney</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
color:white;
background-color: rgb(44, 47, 51);
}

QComboBox QAbstractItemView {
   color:white;
    selection-background-color:rgb(88, 101, 242);
	backgroud-color:rgb(44, 47, 51);
	
}

QComboBox QScrollBar
{
background: #5865F2
}

QComboBox QAbstractItemView QScrollBar
{
background: #5865F2
}



</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dayOfMonthComboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whitney</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
color:white;
background-color: rgb(44, 47, 51);
}

QComboBox QAbstractItemView {
   color:white;
    selection-background-color:rgb(88, 101, 242);
	backgroud-color:rgb(44, 47, 51);
	
}

QComboBox QScrollBar
{
background: #5865F2
}

QComboBox QAbstractItemView QScrollBar
{
background: #5865F2
}



</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whitney</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel[text]{
color: white;}
QLabel{
background-color:rgb(35, 39, 42);
border:none;}</string>
    </property>
    <property name="text">
     <string>Day of the month:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPeriodicallyIncomePushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whitney</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
     background-color: rgb(88, 101, 242);
 }
 QPushButton#pushButton:pressed {
   	background-color: rgb(88, 157, 242); 
 }
 QPushButton#pushButton:hover {
     
	background-color: rgb(88, 157, 242);
 }
 QPushButton[text] {
    color:white;
 }</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelAddIncomePushButton">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>380</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whitney</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
     background-color: rgb(88, 101, 242);
 }
 QPushButton#pushButton:pressed {
   	background-color: rgb(88, 157, 242); 
 }
 QPushButton#pushButton:hover {
     
	background-color: rgb(88, 157, 242);
 }
 QPushButton[text] {
    color:white;
 }</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whitney</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel[text]{
color: white;}
QLabel{
background-color:rgb(35, 39, 42);
border:none;}</string>
    </property>
    <property name="text">
     <string>Card Currency:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cardCurrencyISO">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whitney</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit[text]{
color: white;}
QLineEdit{
background-color: rgb(44, 47, 51);
border:none;}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>ISO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="incomeSourceLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whitney</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit[text]{
color: white;}
QLineEdit{
background-color: rgb(44, 47, 51);
border:none;}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Source</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
